### PR TITLE
add docker and saucelabs Options

### PR DESCRIPTION
**Pull request by rycornell**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
https://www.npmjs.com/package/wdio-sauce-service
https://www.npmjs.com/package/wdio-docker-service
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/webdriverio/index.d.ts [hunk 1]
```diff
@@ -178,6 +178,9 @@ declare namespace WebdriverIO {
         setWindowRect?: boolean;
         timeouts?: Timeouts;
         unhandledPromptBehavior?: string;
+
+        // wdio-sauce-service specific
+        build?: string;
```
Comment (timbru31) [inline types/webdriverio/index.d.ts]: Could you please link to the documentation where build is defined? I can't find any documentation at the https://github.com/webdriverio-boneyard/wdio-sauce-service

Comment (timbru31) [reply types/webdriverio/index.d.ts]: Also, aren't you omitting the `sauceConnectOpts` property?

**Review by timbru31 — CHANGES_REQUESTED** [review]

File: types/webdriverio/index.d.ts (hunk 1 above)
Comment (rycornell) [reply types/webdriverio/index.d.ts]: Thanks, I've added `sacueConnectOpts`.  The documentation isn't explicit about the `build` property, but it was mentioned in this issue: https://github.com/webdriverio-boneyard/wdio-sauce-service/issues/33 which links to this example: https://github.com/webdriverio/webdriverio/blob/master/test/conf/travis-ci.js#L21

**Review by timbru31 — APPROVED** [review]
LGTM

**Pull request merged**